src/actuator.c

```diff
@@ -48,6 +48,22 @@ int nrm_actuator_set_choices(nrm_actuator_t *actuator,
 	return 0;
 }
 
+nrm_string_t nrm_actuator_uuid(nrm_actuator_t *actuator){
+	return actuator->uuid;
+}
+
+nrm_uuid_t *nrm_actuator_clientid(nrm_actuator_t *actuator){
+	return actuator->clientid;
+}
+
+double nrm_actuator_value(nrm_actuator_t *actuator){
+	return actuator->value;
+}
+
+nrm_vector_t *nrm_actuator_choices(nrm_actuator_t *actuator){
```

Comment:
the easiest way to deal with this one is to take a `nrm_vector_t *` as an argument, and fill it out, like we do for nrm_client_list_sensors. Returning the field here makes the python side messier.